dis.h

```diff
@@ -73,6 +73,7 @@ extern struct word_format pt_word_format;
 extern struct word_format tape_word_format;
 extern struct word_format tape7_word_format;
 extern struct word_format x_word_format;
+extern struct word_format oct_word_format;
```

Comment:
Picky comment: this list and the one in word.c are in sorted order (so the formats are also sorted in the help message).

Reply:
Thanks, good catch.